src/javascript/EditPanelContainer/EditPanel/EditPanel.utils.js

```diff
@@ -23,10 +23,18 @@ export function getPropertiesToMutate(nodeData = {}, formValues = {}, fields = [
 
                 if (field.formDefinition.multiple) {
                     valueObj.values = value;
+
+                    if (fieldType === 'DATE') {
+                        valueObj.formattedDateValues = value;
+                    }
```

Comment:
We need to use the date format to parse this value and transform it as date, note that the server must use the date format to send the date